ui/src/app/views/workspace/component.ts

```diff
@@ -81,6 +82,10 @@ export class WorkspaceComponent implements OnInit {
                 this.cohortList.push(coho);
               }
               this.cohortsLoading = false;
+            },
+            error => {
+              this.cohortsLoading = false;
```

Comment:
Does this have an `always` option since you're doing `this.cohortsLoading = false` no matter what?